src/client/src/ui/blotter/components/BlotterGrid.tsx

```diff
@@ -1,18 +1,18 @@
 import { styled } from 'rt-util'
 
 export default styled('div')`
-  height: calc(100% - 70px);
+  height: calc(100% - 76px);
   background-color: ${({ theme: { background } }) => background.backgroundSecondary};
   border-radius: 3px;
 
   .ag-header {
     border-bottom: 2px solid ${({ theme: { background } }) => background.backgroundPrimary};
```

Comment:
In general, try to avoid using `border` — they're hard to account for in the vertical rhythm.

The same could be achieved by
```css
box-shadow: 0 -2px 0 ${({ theme: { background } }) => background.backgroundPrimary} inset;
```
Then the vertical space needed can be accounted for within the own element.

Of course, you may also need the border to not be `inset` — and we'll assume the user accounts for the space, if needed.

Reply:
If the border does not look properly centered, there are some cases where you may split the definition between an `inset` and external box-shadow.

Reply:
You'll also notice that I did specify the size as a `rem` value — this will depend on the visual intent.